README.md

```diff
@@ -57,6 +57,12 @@ rtc_build_examples = false
 target_cpu = "x64"
 is_debug = false
 ````
+- If you wish to use the Intel GPRA BWE library, you need to add three additional variables, shown below. The lib root should be the directory that contains `gpra_lib.lib`, and the header root should be directory that contains `grpa_bwe.h`. Using the // syntax, the path is relative to your `src` directory.
```

Comment:
why not allowing using to specify an absolute path?

Reply:
Maybe you can use absolute syntax, this is just what I've tested. I don't know much about gn/ninja... I'm just trying to document what I know so that someone else doesn't have to waste time experimenting.